admin-dev/themes/new-theme/js/pages/product/edit/combinations-manager.js

```diff
@@ -98,24 +125,62 @@ export default class CombinationsManager {
     const {tokenKey} = ProductMap.combinations.combinationItemForm;
 
     new SubmittableInput(ProductMap.combinations.quantityInputWrapper, async (input) => {
-      await this.combinationsService.updateListedCombination(
-        this.findCombinationId(input),
-        {[quantityKey]: input.value, [tokenKey]: combinationToken},
-      );
+      await this.combinationsService.updateListedCombination(this.findCombinationId(input), {
+        [quantityKey]: input.value,
+        [tokenKey]: combinationToken,
+      });
     });
 
     new SubmittableInput(ProductMap.combinations.impactOnPriceInputWrapper, async (input) => {
-      await this.combinationsService.updateListedCombination(
-        this.findCombinationId(input),
-        {[impactOnPriceKey]: input.value, [tokenKey]: combinationToken},
-      );
+      await this.combinationsService.updateListedCombination(this.findCombinationId(input), {
+        [impactOnPriceKey]: input.value,
+        [tokenKey]: combinationToken,
+      });
     });
 
     new SubmittableInput(ProductMap.combinations.referenceInputWrapper, async (input) => {
-      await this.combinationsService.updateListedCombination(
-        this.findCombinationId(input),
-        {[referenceKey]: input.value, [tokenKey]: combinationToken},
-      );
+      await this.combinationsService.updateListedCombination(this.findCombinationId(input), {
+        [referenceKey]: input.value,
+        [tokenKey]: combinationToken,
+      });
+    });
+  }
+
+  /**
+   * @private
+   */
+  initSortingColumns() {
+    this.$combinationsContainer.on('click', ProductMap.combinations.sortableColumns, (event) => {
+      const $sortableColumn = $(event.currentTarget);
+      const columnName = $sortableColumn.data('sortColName');
+
+      if (!columnName) {
+        return;
+      }
+
+      let direction = $sortableColumn.data('sortDirection');
+
+      if (!direction || direction === 'desc') {
+        direction = 'asc';
+      } else {
+        direction = 'desc';
+      }
+
+      // Reset all columns, we need to force the attributes for CSS matching
+      $(ProductMap.combinations.sortableColumns, this.$combinationsContainer).removeData('sortIsCurrent', undefined);
+      $(ProductMap.combinations.sortableColumns, this.$combinationsContainer).removeData('sortDirection', undefined);
```

Comment:
```suggestion
      $(ProductMap.combinations.sortableColumns, this.$combinationsContainer).removeData('sortIsCurrent');
      $(ProductMap.combinations.sortableColumns, this.$combinationsContainer).removeData('sortDirection');
```
This function expect one parameter :thinking:

Reply:
Yep my bad, it used to be `setData` I forgot to remove the second parameter